spartan/aztec-network/values/exp-1.yaml

```diff
@@ -0,0 +1,56 @@
+network:
+  public: false
+
+aztec:
+  slotDuration: 36
+  epochDuration: 32
+  realProofs: false
+
+images:
+  aztec:
+    pullPolicy: Always
+
+telemetry:
+  enabled: true
+  otelCollectorEndpoint: http://35.197.100.168:4318
```

Comment:
It should be fine to use dns

Reply:
I didn't know the otel collector had a dns name?